go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	k8s.io/kubernetes v1.23.0
 	sigs.k8s.io/yaml v1.2.0
 	stathat.com/c/consistent v1.0.0
-	volcano.sh/apis v0.0.0-20211109104455-7e8d893a9f66
+	volcano.sh/apis v1.5.0-beta.0.0.20220112022954-fd6c584c7994
```

Comment:
I'm not sure if this version number is spelled correctly, it seems to be different than in another pr

https://github.com/volcano-sh/volcano/pull/1964/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R30

Reply:
Yes, I've updated `apis` vendor to latest master branch.